content/applications/marketing/marketing_automation.rst

```diff
@@ -1,121 +1,186 @@
+the template cards to open the campaign form.
 
-To add a more specific filter to a marketing automation campaign, click the :guilabel:`Add
-condition` button in the :guilabel:`Filter` field. Doing so reveals a series of other configurable
-filter rule fields.
+.. tip::
+   To display the campaign template cards again after a campaign has been created, type the name of
+   a campaign that does not exist in the database into the :guilabel:`Search...` bar, then press
+   :kbd:`Enter`.
 
-In the rule fields, customizable equations can be configured for Odoo to use when filtering who to
-include or exclude in this specific marketing campaign.
+   For example, searching for `empty` displays the campaign template cards again as long as there is
+   not a campaign with the name "empty" in the database.
 
-.. image:: marketing_automation/filter-node-equation-fields.png
-   :align: center
-   :alt: How the filter rule equation fields look in Odoo Marketing Automation campaigns.
+Targets & filters
+=================
 
-.. note::
-   :guilabel:`Records` refer to contacts in the system that fit the specified criteria for a
-   campaign.
+The *Target* and *Filter* section of the campaign form, also referred to as the domain, contains the
+fields used to define the target audience for the campaign's reach (i.e., records).
 
-Also, once :guilabel:`Add condition` is clicked, the ability to :guilabel:`Save as Favorite Filter`
-becomes available on the campaign form.
+Records
+-------
 
-There is also the option to match records with :guilabel:`all` or :guilabel:`any` of the rules
-configured in the :guilabel:`Filter` field.
+The contacts in the system that fit the specified criteria for a campaign are referred to as
+*records*.
 
-To choose either of those options, click :guilabel:`all` from the middle of the sentence
-":guilabel:`Match records with all of the following rules`" to reveal a drop-down menu with those
-options.
+The number of records that are displayed next to the campaign *Filter* represents the total number
+of records the campaign is targeting.
 
-.. image:: marketing_automation/match-all-any-rules-drop-down.png
-   :align: center
-   :alt: Match records with all or any of the rules in Filter field for marketing campaigns.
+Participants
+------------
 
-When the first field of the rule equation is clicked, a nested drop-down menu of options appears on
-the screen where specific criteria is chosen based on needs of the campaign.
+The records that are engaged by the campaign are referred to as *participants*.
 
-The remaining fields on the rule equation further define the criteria, which is used to determine
-which records in the database to include or exclude in the execution of the campaign.
+The number of participants engaged in a test run are shown in the *Tests #* smart button. The number
+of participants engaged in a running or stopped campaign are shown in the *Participants #* smart
+button.
```

Comment:
Thanks @meng-odoo! The **Tests** or **Participants** smart buttons only show after a test has ran or the campaign has started, which is why I chose not to use :guilabel:s here because I have not instructed to perform a test or start the campaign, but rather stating these buttons appear when xyz happens, so these are not part of the active instruction or immediately visible on the UI.

I can rework this sentence 👍